README.md

```diff
@@ -1,2 +1,70 @@
-# .github
-Organization-wide repository for common governance documents.
+# Notary Project Overview
+
+The Notary Project is a set of specifications and tools intended to provide a cross-industry standard for securing software supply chains. 
```

Comment:
'securing software supply chains' is a very broad statement that implies considerably more capabilities than this project is intending to address. Would recommend modifying this statement to focus more around a standard for securing software supply chains through image signing, verification, and key management (? while the updated text below  provides the history and describes the challenges or limitations with the previous instance of Notary, it does not define the capabilities of each and whether they have changed.)

Reply:
Drawing from a call on the WG-Artifacts, my understanding was that we should keep the organization charter broad enough so that for incremental changes the charter need not be revised. calling out the scope is fair

something along the lines - 
> The Notary Project is a set of specifications and tools intended to provide a cross-industry standard for securing software supply chains through signing and verification" 

The projects do differ in key management scenarios is my understanding.

Reply:
Updated the text